src/app/pages/SearchResultsPage/useRedirectIfSingleResult.ts

```diff
@@ -34,7 +34,7 @@ export function useRedirectIfSingleResult(
         redirectTo = RouteUtils.getTransactionRoute(item, item.eth_hash || item.hash)
         break
       case 'account':
-        redirectTo = RouteUtils.getAccountRoute(item, (item as RuntimeAccount).address_eth ?? item.address)
+        redirectTo = `${RouteUtils.getAccountRoute(item, (item as RuntimeAccount).address_eth ?? item.address)}?q=${searchTerm}`
```

Comment:
For future improvement:
https://92fdad12.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qp000ff4weh6t6083hazdpw0r902leffp5gjgu55?q=oasis1qp000ff4weh6t6083hazdpw0r902leffp5gjgu55

Reply:
Up to now, the idea was that we only do highlighting when (partial) text matching is involved. It didn't occur to me that we could just as well highlight fields when equality was for testing, such as block number, tx hash or address. Indeed, we could do that. I will discuss this with @donouwens.

Reply:
I didn't mean that :D I meant: I prefer the old URLs when searching for an exact address

Reply:
> I prefer the old URLs when searching for an exact address

Oh, that's not difficult to change. Fixed that now.